server/cmd/mmctl/commands/user_e2e_test.go

```diff
@@ -736,13 +736,15 @@ func (s *MmctlE2ETestSuite) TestDeleteUsersCmd() {
 		s.Len(printer.GetLines(), 1)
 		s.Len(printer.GetErrorLines(), 0)
 
-		deletedUser := printer.GetLines()[0].(*model.User)
-		s.Require().Equal(newUser.Username, deletedUser.Username)
+		// The output is now a job creation confirmation with job ID and username
+		jobResult := printer.GetLines()[0].(struct {
+			Id       string
+			Username string
+		})
+		s.Require().Equal(newUser.Username, jobResult.Username)
+		s.Require().NotEmpty(jobResult.Id) // Job ID should be present
 
-		// expect user deleted
-		_, err = s.th.App.GetUser(newUser.Id)
-		s.Require().NotNil(err)
-		s.Require().Equal("GetUser: Unable to find the user., resource \"User\" not found, id: "+newUser.Id, err.Error())
+		// User deletion is now asynchronous, so we don't check immediate deletion
```

Comment:
This feels like somewhat of a gap, though. Could we accelerate the job getting picked up and wait for its success (e.g. simple polling up to a timeout)?

Reply:
Thanks, done.